input/pages/app-launch.md

```diff
@@ -109,9 +109,27 @@ for example:
 - App is an HTML5 or JS in-browser app (including single-page applications) that would expose the secret in user space
 - App is a native app that can only distribute a secret statically
 
+#### Validating the state parameter
+All SMART apps SHALL validate the state for any request sent to its redirect URL. This is to combat Cross-Site Request Forgery (CSRF) attacks as outlined in [RFC6819 section 4.4.1.8](https://datatracker.ietf.org/doc/html/rfc6819#section-4.4.1.8).
+
+In general, validating the state requires:
+1. Generating a `state_verifier`:`state` and pair
+2. Storing the `state_verifier` in a pre-login session
```

Comment:
I'm confused about what `state_verifier` is here. What does it mean to generate one, and what does it mean to "check the presented `state` against" one?

I don't see this term used in the referenced documents. I would have though a typical pattern would be just to store the expected state value in a session (e.g., for an HTML5 app, `sessionStorage.expectedState = ...`)